src/components/Articles/index.tsx

```diff
@@ -1,80 +1,73 @@
-import styles from "./styles.module.css";
+
+import styles from "./styles.module.scss";
+import React from "react";
 
 const Articles = () => {
     return(
-        <div>
-            <div className={styles.text}>VOLÚMEN</div>
+        <div >
+            <div className={styles.titulo}>VOLÚMENES</div>
```

Comment:
this is bad this information should have come from a mock file, or simulated the data from another placer as https://jsonplaceholder.typicode.com or something like that.